content/appendices.md

```diff
@@ -556,7 +556,7 @@ The `domain` of a user ID is the [server name](#server-name) of the
 homeserver which allocated the account.
 
 The length of a user ID, including the `@` sigil and the domain, MUST
-NOT exceed 255 characters.
+NOT exceed 255 bytes.
```

Comment:
For future reference: this restriction comes from the fact that a user ID must fit inside a state key (in the `m.room.member` event), and state keys have always been limited to 255 bytes (since https://github.com/matrix-org/matrix-spec/commit/6a0595bc46c3751090d5d24e2ab94647c9573915)

Reply:
no, that's fine. I just wanted to make a record so that when, in a few months, we come to look at this PR, we remember the thought process.